public/js/controllers/proposal_controller.js

```diff
@@ -1,78 +1,173 @@
 import { Controller } from 'stimulus'
 import { getDefault } from '../helpers/module_helper'
+import { multiColumnBarPlotter, synchronize } from '../helpers/chart_helper'
+import axios from 'axios'
 
+let common = {
+  labelsKMB: true,
+  legend: 'always',
+  fillGraph: true,
+  strokeWidth: 2,
+  labelsUTC: true,
+  gridLineColor: '#C4CBD2',
+  legendFormatter: legendFormatter
+}
+
+let percentConfig = {
+  ...common,
+  labels: ['Date', 'Percent Yes'],
+  ylabel: 'Percentage Yes Votes (%)',
+  colors: ['#2971FF'],
+  labelsSeparateLines: true,
+  underlayCallback: function (canvas, area, g) {
+    let xVals = g.xAxisExtremes()
+    let xl = g.toDomCoords(xVals[0], 60)
+    let xr = g.toDomCoords(xVals[1], 60)
+
+    canvas.beginPath()
+    canvas.strokeStyle = '#ED6D47'
+    canvas.moveTo(xl[0], xl[1])
+    canvas.lineTo(xr[0], xr[1])
+    canvas.closePath()
+    canvas.stroke()
+  }
+}
+
+let cummulativeConfig = {
+  ...common,
+  labels: ['Date', 'Total Votes'],
+  ylabel: 'Cummulative Votes',
```

Comment:
Not sure about this.

Reply:
It's pretty clear it's not new votes per block.  Instead of "cumulative votes" I'd think a more plain-language phrase like "total votes" or "vote count" would be clearer.